Tools/machines/frontier-olcf/frontier_warpx.profile.example

```diff
@@ -0,0 +1,55 @@
+# please set your project account
+#export proj=APH114-frontier
+
+# required dependencies
+module load cmake/3.22.2
+module load craype-accel-amd-gfx90a
+module load rocm/5.1.0
+module load cray-mpich
+#module load cce/14.0.0  # must be loaded after rocm
```

Comment:
They have adios2/2.8.1 available if we have cce/14.0.0 loaded. There is an issue with the latest CCE 14.0.1 that causes some builds to fail so a full software stack is not available for that compiler toolchain. CCE 14.0.0, however, has the latest versions of most E4S packages.

Reply:
Due to a breaking change in read of operators, we would prefer adios/2.8.2 (or 2.7.1)